kolibri/plugins/facility/assets/src/modules/manageCSV/actions.js

```diff
@@ -79,12 +79,12 @@ function getExportedCSVsInfo(store) {
     const data = response.data;
     if (data.session != null) {
       getCSVLogRequest(store, 'session', store.rootGetters.activeFacilityId);
-      const sessionTimeStamp = new Date(data.session * 1000);
```

Comment:
What's the reason for these changes?

Reply:
In the facility.views we used to get the time with `os.path.getmtime(log_path)` but I used the Storage class's `get_modified_time(log_path).isoformat()`. I saw that same `.isoformat()` being used elsewhere in `exported_csv_info` and it jived well when i removed the multiplication here.